clang/lib/CodeGen/CGCleanup.cpp

```diff
@@ -1118,6 +1118,8 @@ void CodeGenFunction::EmitBranchThroughCleanup(JumpDest Dest) {
 
   // Create the branch.
   llvm::BranchInst *BI = Builder.CreateBr(Dest.getBlock());
+  // This is the primary instruction for this atom, acting in place of a ret.
```

Comment:
I think in isolation this comment will be hard to understand, particularly relating it to a `ret`. Perhaps "this atom" -> "the source atom leaving this scope"?